src/classes/timer_label.vala

```diff
@@ -230,11 +236,36 @@ namespace pdfpc {
                 if (this.time < this.duration) {
                     timeInSecs = duration - this.time;
                     // Still on presentation time
-                    if (timeInSecs < this.last_minutes * 60)
-                        context.add_class("last-minutes");
+                    if (Options.timer_pace_color) {
```

Comment:
Please remove all trailing white space.

Reply:
Sorry about it, done.

Also, while looking closely, it appeared there is no need to pass to getTimerLabel() most of its arguments, as they are available from Options. So I made the respective changes, hope it is fine with you.

Reply:
Thanks for your approval, I'm really flattered to see you suggest making this default. However, knowing how picky people are about changing default settings, I think it might be worth leaving it as an option for another release. But of course, it is completely up to you as a maintainer.